workflow/config/empirical_pred_config.yaml

```diff
@@ -20,8 +20,7 @@ ts_inference:
    ancestral_states: "empirical_prediction/test_data/test-AA.npy"
 
 metadata:
-  populations: 
-    pop1: 10
+  sample_popns: "empirical_prediction/test_data/test_sample_popns.csv"
```

Comment:
looks good-- what about sticking this as "sample_population" field in the preceeding `ts_inference` section? Rather than having `metadata` section.

Reply:
good idea

Reply:
looks good to merge to me